### DISCUSSION TITLE
Annoying Unsponsored Label

### DISCUSSION THREAD

**Discussion by antonreshetov** [massCodeIO/massCode#160] [Announcements]
Starting with version 2.10.0 there is a "Unsponsored" label, which will probably annoy most people. I used the Sublime model as the basis. 

The good news is that there is a separate build for everyone who supports (donations) massCode without this text and notifications. 

Of course because it's an OSS, anyone can look around in the code and remove this label and make their own build. This is not prohibited, but I think it would be at least unethical. But I took care of those who supported the project. Write me in [DM](https://twitter.com/anton_reshetov) with your GitHub account and I'll invite you to the private repo. Also, all contributors will also be invited to this repo. 

Thank you for understanding.

**Comment by v-c0de** [comment]
hi @antonreshetov - by support, do you mean donations and code contributors only?

**Comment by antonreshetov** [comment]
> hi @antonreshetov - by support, do you mean donations and code contributors only? 

Evaluating other support is much more difficult. Do you have something specific in mind?

**Reply by v-c0de** [reply]
I guess in my case, trying to be involved in your project through discussions here, suggestions, obviously starring the project early on and spreading the word to my friends and colleagues (reporting bugs if/where found - though I haven't flagged any so far that others didn't already pick up on)

**Reply by antonreshetov** [reply]
> I guess in my case, trying to be involved in your project through discussions here, suggestions, obviously starring the project early on and spreading the word to my friends and colleagues (reporting bugs if/where found - though I haven't flagged any so far that others didn't already pick up on) 

I certainly appreciate everyone trying to make improvements, through ideas. Thank you. You share ideas or find bugs, I implement them or fix them, I think we at least repay each other. But at the same time I created a project that you use, which is useful to you, I think. So I would not equate ideas, bug reports and donations.

**Reply by v-c0de** [reply]
yeah that's fair. Again, was just trying to clarify what you meant by 'support', because you also have the below on your homepage: 
![image](https://user-images.githubusercontent.com/65813722/180252490-48d82bc7-7dca-4bcc-8158-67820bef72ab.png)

**Reply by antonreshetov** [reply]
With your permission, I added in the parentheses that I meant donations.

I apologize if I confused you.

**Reply by v-c0de** [reply]
of course mate, no problem! It's a free product you've made that's actually good, so no complaints here :)

**Comment by MarcelBigger** [comment]
Certainly will support. Those who installed the Windows-build have to do what?

**Comment by pa-0** [comment]
Hi Anton! I tried to follow your link to X (twitter) to ask for invite to private repo. Unfortunately, it looks like only 'verified users' can DM.

**Reply by antonreshetov** [reply]
You can always reach me at [email](mailto:reshetov.art@gmail.com)
